content/en-us/reference/engine/classes/Debris.yaml

```diff
@@ -99,74 +97,32 @@ properties:
 methods:
   - name: Debris:AddItem
     summary: |
-      Allows the developer to schedule the removal of the object without
-      yielding any code.
+      Registers a given `Class.Instance` for removal within the specified
```

Comment:
The previous summary sounded more like a summary for the Debris service in general, and not specifically for AddItem().

Reply:
Yeah it was weird, the entire service summary was duplicated in the AddItem summary.